src/components/index.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+export { default as Account } from './Account';
```

Comment:
IMHO the entire file doesn't really add value.

Reply:
The actual case is that it makes the export of components to be done straight from the components dir. so instead of:
`import Account from './components/Account';`
it helps to change to:
`import { Account } from './components';`
The example above may look not-much-diff but in case of multiple imports, that reduces the lines and readability.

If though this is not a preferable way I can just rollback - just let me know.